app/assets/src/components/common/DetailsSidebar/GeneDetailsMode/GeneDetailsMode.jsx

```diff
@@ -65,10 +67,13 @@ export default class GeneDetailsMode extends React.Component {
 
   async getGeneInfo(geneName) {
     const ontology = await getOntology(geneName);
-    const cardEntryFound = ontology.error !== "" ? false : true;
+    const ontologyInfoFound = ontology.error === "";
+    // either the CARD accession is 7 or it is useless
```

Comment:
Maybe change this comment to `All CARD accession ids are 7 in length. Check if a card entry was found.`